### ISSUE TITLE
[Refactor]: Audio visualizer component

### ISSUE THREAD

**Issue by CaseRegan** [Amijakan/mochatto#240]
### Refactor Request

Move the audio visualizer found on the join page into its own component. Make it more extensible by making it easy to set a max width and have the gain display as a percentage of the max width, rather than a raw pixel number.

**Comment by siberowl** [comment]
I agree since there's also another class called AudioVisualizer.ts for visualizing the voice volume as a circle around the avatar, probably should name it something more specific like AudioVisualizerBar (and maybe the class like AudioVisualizerCircle)

**Comment by Njima1572** [comment]
I'd also like multiplier not being transported over datachannel. This could be calculated locally, and feels like it is generating unnecessary data transport / render.
